cli/command/context/list.go

```diff
@@ -0,0 +1,100 @@
+package context
+
+import (
+	"fmt"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/command/formatter"
+	"github.com/docker/cli/cli/context/docker"
+	kubecontext "github.com/docker/cli/cli/context/kubernetes"
+	"github.com/docker/cli/kubernetes"
+	"github.com/spf13/cobra"
+)
+
+type listOptions struct {
+	format string
+	quiet  bool
+}
+
+func newListCommand(dockerCli command.Cli) *cobra.Command {
+	opts := &listOptions{}
+	cmd := &cobra.Command{
+		Use:     "ls [OPTIONS]",
+		Aliases: []string{"list"},
+		Short:   "List contexts",
+		Args:    cli.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runList(dockerCli, opts)
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.StringVar(&opts.format, "format", formatter.TableFormatKey, "Pretty-print contexts using a Go template")
```

Comment:
nit (for a follow-up); for other commands we use `""` as default, and in the code use something like;

```
	format := opts.format
	if len(format) == 0 {
		format = formatter.TableFormatKey
	}
```

e.g. https://github.com/docker/cli/blob/a900ba8aeff764a5fb5bb7e18eee25d75a220732/cli/command/system/df.go#L45-L48


We can do a follow-up to decide which approach to take for all (either way makes some sense)

Reply:
ok for a followup